common/match/create.ts

```diff
@@ -91,3 +94,33 @@ export async function createMatch(pupil: Pupil, student: Student, pool: Concrete
 
     logger.info(`Created Match(${match.uuid}) for Student(${student.id}) and Pupil(${pupil.id})`);
 }
+
+export async function createAdHocMeeting(matchId: number, user: User) {
+    const match = await getMatch(matchId);
+    const { pupilId, studentId } = match;
+
+    const pupil = await getPupil(pupilId);
+    const student = await getStudent(studentId);
+
+    const appointment: AppointmentCreateMatchInput[] = [
+        {
+            title: `Sofortbesprechung - ${pupil.firstname} und ${student.firstname} `,
+            matchId: matchId,
+            start: new Date(),
```

Comment:
This could cause time issues again, because we're creating a date in the backend which runs in UTC.